pkgs/development/tools/wabt/default.nix

```diff
@@ -0,0 +1,65 @@
+{ stdenv, fetchFromGitHub, cmake, python3 }:
+
+stdenv.mkDerivation rec {
+  name = "wabt-${version}";
+  version = "1.0.5";
+
+  src = fetchFromGitHub {
+    owner  = "WebAssembly";
+    repo   = "wabt";
+    rev    = version;
+    sha256 = "1cbak3ach7cna98j2r0v3y38c59ih2gv0p6f43qp782pyj07hzfy";
+  };
+
+  nativeBuildInputs = [ cmake ];
+  cmakeFlags = [ "-DBUILD_TESTS=OFF" ];
+  buildInputs = [ python3 ];
+
+  meta = with stdenv.lib; {
+    description = "The WebAssembly Binary Toolkit";
+    longDescription = ''
+      WABT (we pronounce it "wabbit") is a suite of tools for WebAssembly,
+      including:
+      <itemizedlist>
```

Comment:
Is it valid to use docbook formatting here? Where does it get rendered in docbook format? I only see one other package `pg_top` that uses <itemizedlist> in its description.

Reply:
Oh indeed! I was confusing `longDescription` with module options. Pushing an updated version, thanks!